packages/pageflow/src/frontend/VideoPlayer/Lazy.js

```diff
@@ -8,9 +7,11 @@ import {assetUrls} from '../assetUrls';
 // A proxy which lazily initializes the real video player.
 export const Lazy = function(template, options) {
   var placeholder = $('<span class="video_placeholder" />'),
-      that = this,
-      readyCallbacks = new $.Callbacks(),
-      disposeTimeout, videoTag, videoPlayer, html;
+    that = this,
+
+    // TODO: Replace $.Callbacks
+    readyCallbacks = new $.Callbacks(),
+    disposeTimeout, videoTag, videoPlayer, html;
```

Comment:
`VideoPlayer.Lazy` is [only used by linkmap page](https://github.com/codevise/pageflow-linkmap-page/blob/c598a8b3775aab41b802f3f976ce1e48f63fe2a0/app/assets/javascript/pageflow/linkmap_page/page_type.js#L203). For video page types it was replaced by a React implementation.

I therefore suggest moving it to `pageflow-paged`. Then it can keep it's jQuery dependency and we do not have to think about `$.Callbacks`.